### ISSUE TITLE
Feature request: store SSH public keys in AD and set passwordless SSH to HeadNode

### ISSUE THREAD

**Issue by rvencu** [aws/aws-parallelcluster#4161]
I found some articles dealing with this as to describe what I am after:

https://access.redhat.com/solutions/5353351

https://blog.laslabs.com/2016/08/storing-ssh-keys-in-active-directory/

**Comment by hanwen-pcluste** [comment]
Thank you for the suggestion. I marked it as a feature request. We will consider this within our team.

**Comment by rvencu** [comment]
Yes, the redhat article is good, I implemented the instructions step by step, and everything works as expected.

Added that as a post-install script https://github.com/rvencu/1click-hpc/blob/main/modules/02.activate.passwordless.AD.headnode.sh
